### ISSUE TITLE
Allow empty password from cmdline

### ISSUE THREAD

**Issue by pfsen** [jfjallid/go-shareenum#2]
Right now it is possible to authenticate using empty password by simply pressing enter when prompted.

```powershell
PS C:\> .\go-shareenum -host 10.10.11.175 -user bobby -enum
Enter password:
2023/07/04 21:52:23 [Notice] [-] Signing is required
2023/07/04 21:52:23 [Notice] [+] Login successful
```

providing the `-pass ''` param with an empty string does not work however.

**Comment by pfsen** [comment]
smbclient implements this with the `-N / --nopass` flag.

**Comment by jfjallid** [comment]
Fixed in commit 948f327ee299058cc730552b2adc996dd4e59f59
